File: OnlineShopAPI/AddressMicroservice/Address.Api/Models/UpdateAddressRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Address.Api.Models
{
    public class UpdateAddressRequest
    {
        public long AddressId { get; set; }

        public string NewCustomer { get; set; }

        public string NewUnitStreet { get; set; }

        public string NewCity { get; set; }

        public string NewState { get; set; }

        public string NewZipcode { get; set; }
    }
}

File: OnlineShopUI/OnlineShop/src/app/model/requests/update-address-request.ts
export class UpdateAddressRequest {
    addressId: number;
    newCustomer: string;
    newUnitStreet: string;
    newCity: string;
    newState: string;
    newZipcode: string;

    constructor(addressId: number, newCustomer: string, newUnitStreet: string, newCity: string,
        newState: string, newZipcode: string){
        this.addressId = addressId;
        this.newCustomer = newCustomer;
        this.newUnitStreet = newUnitStreet;
        this.newCity = newCity;
        this.newState = newState;
        this.newZipcode = newZipcode;
    }
}
